rpadmin/admin.go

```diff
@@ -271,7 +273,7 @@ func (a *AdminAPI) ForHost(url string) (*AdminAPI, error) {
 }
 
 func (a *AdminAPI) newAdminForSingleHost(host string) (*AdminAPI, error) {
-	return newAdminAPI([]string{host}, a.auth, a.tlsConfig, nil, a.forCloud)
+	return newAdminAPI([]string{host}, a.auth, a.tlsConfig, a.dialer, a.forCloud)
```

Comment:
Any chance we could sneak in a regression test for this?

Reply:
done